specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2023-12-10-preview/cloudhsm.json

```diff
@@ -0,0 +1,967 @@
+        }
+      },
+      "description": "Properties of a Cloud HSM Cluster.",
+      "type": "object"
+    },
+    "CloudHsmClusterPatchParameters": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "The Cloud HSM Cluster's tags",
+          "additionalProperties": {
+            "type": "string",
+            "description": "The Cloud HSM Cluster's tag's additional properties"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/CloudHsmClusterSku",
+          "description": "SKU details"
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v5/managedidentity.json#/definitions/ManagedServiceIdentity"
+        }
+      },
+      "description": "Patchable properties of the Cloud HSM Cluster",
+      "type": "object"
+    },
+    "CloudHsmClusterSecurityDomainProperties": {
+      "type": "object",
+      "description": "Security domain properties information for Cloud HSM cluster",
+      "properties": {
+        "fipsState": {
+          "format": "int32",
+          "type": "integer",
+          "description": "FIPS state information for security domain"
+        },
+        "activationStatus": {
+          "type": "string",
+          "description": "status of security domain activation"
+        }
+      }
+    },
+    "CloudHsmClusterSku": {
+      "type": "object",
+      "description": "Cloud Hsm Cluster SKU information",
+      "required": [
+        "name",
+        "family"
+      ],
+      "properties": {
+        "family": {
+          "type": "string",
+          "description": "Sku family of the Cloud HSM Cluster",
+          "enum": [
+            "B"
+          ],
+          "x-ms-client-default": "B",
+          "x-ms-enum": {
+            "name": "CloudHsmClusterSkuFamily",
+            "modelAsString": true
+          }
+        },
+        "name": {
+          "type": "string",
+          "description": "Sku name of the Cloud HSM Cluster",
+          "enum": [
+            "Standard_B1",
+            "Standard B10"
+          ],
+          "x-ms-enum": {
+            "name": "CloudHsmClusterSkuName",
+            "modelAsString": false
+          }
+        },
+        "capacity": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Sku capacity"
+        }
+      }
+    },
+    "CloudHsmClusterBackupProperties": {
+      "type": "object",
+      "description": "Cloud Hsm Cluster backup information",
+      "properties": {
+        "storageResourceUri": {
+          "type": "string",
+          "description": "Azure storage account to store the backup files"
+        },
+        "lastUpdatedDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Last Date Time that Customer Enabled Backup was taken"
+        },
+        "backupStatus": {
```

Comment:
I believe this is not needed since we are only reporting last successful backup time above. I dont think HFC records failed backup jobs. Let's sync back on this.

Reply:
Removed from the model.